lib/internal/errors.js

```diff
@@ -1036,6 +1036,11 @@ E('ERR_ENCODING_INVALID_ENCODED_DATA', function(encoding, ret) {
 }, TypeError);
 E('ERR_ENCODING_NOT_SUPPORTED', 'The "%s" encoding is not supported',
   RangeError);
+E('ERR_ESM_LOADER_REGISTRATION_UNAVAILABLE', 'Programmatically registering custom ESM loaders ' +
+  'currently requires at least one custom loader to have been registered via ' +
+  'the --experimental-loader flag. A no-op loader registered via CLI is sufficient ' +
+  '(ex: `--experimental-loader "data:text/javascript,"` do NOT omit the ' +
+  'trailing comma).', Error);
```

Comment:
Do we plan on keeping this error around? If not, can we re-use a more generic error code?

Reply:
What do you mean with a more generic error code?

Reply:
No, it's temporary. I just looked through the existing ones, but none seem applicable (maaaaybe [`ERR_MISSING_OPTION `](https://nodejs.org/api/errors.html#err_missing_option)).